src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -106,7 +106,50 @@ private void updateHearing(HearingWrapper wrapper) throws UpdateCaseException, I
                 response.toString());
 
         hearingResponseUpdate(wrapper, response);
+    }
+
+    private HmcUpdateResponse createHmcUpdateResponse(HearingWrapper wrapper) throws InvalidMappingException {
+        HearingRequestPayload hearingPayload = buildHearingPayload(wrapper, referenceDataServiceHolder);
+        String hearingId = getHearingId(wrapper);
```

Comment:
This will just return null when create hearing is called

Reply:
Agree will fix it